scripts/index.html

```diff
@@ -34,14 +34,14 @@ <h1 id="page-title">
         <div style="display: flex; justify-content: space-between;">
           <div>
             <h2>Local Stream:</h2>
-            <audio id="local-audio" autoplay playsinline></audio>
+            <!-- <audio id="local-audio" autoplay playsinline></audio> -->
             <video style="width:25rem" id="local-video" autoplay playsinline></video>
           </div>
           <h2 id="meeting-title"></h2>
           <div>
             <h2>Remote Stream:</h2>
             <audio id="remote-audio" autoplay playsinline></audio>
-            <video style="width:25rem" id="remote-video" autoplay playsinline></video>
+            <video style="width:25rem" id="remote-video" autoplay playsinline muted></video>
```

Comment:
Shouldn't `muted` be on the audio track? Or does the video have its own audio track?